### DISCUSSION TITLE
Github Education pack approved but still shown "your benefits will be available soon"

### DISCUSSION THREAD

**Discussion by abdelghaniBensalih** [community/community#137363] [GitHub Education] [answered]
### Hi everyone,

Hello, GitHub Education Community!
This is my first time to be here in the discussion. I hope you all are doing fine. I'm inquiring about my application for the GitHub education pack. I already got approved but still pending "will be available soon," and I still want to apply for the github student ambassador. I really loved what it offers and how it can shape my school community and me, of course, and the close date is going to be 1:59:00 pm Pacific Time on August 31, 2024, so close. Hope to be accepted soon.

I would like to hear your feedback; thank you!

**Comment by davevad93** [comment, marked as answer]
Hi @abdelghaniBensalih ,

I suppose you're talking about [applying to be a GitHub Campus Expert](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert), applications will close on Sunday September 1 at 8:59 AM GMT+2 , so unfortunately I don't know if your benefits will be available before that date because it usually takes a few days and sometimes in periods with high volume of request it can take even longer like a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @abdelghaniBensalih was approved and your benefits are now available.`

However, applications for the program will open again in February, so you'll have another opportunity to apply in a few months.

**Reply by abdelghaniBensalih** [reply]
Hello @davevad93 thank you  so much! Your answer was very helpful!

**Reply by davevad93** [reply]
You're welcome @abdelghaniBensalih 👍
